API.md

```diff
@@ -3015,7 +3017,7 @@ Each request object includes the following properties:
   different response. Contains `null` when no response has been set (e.g. when a request terminates
   prematurely when the client disconnects).
 - `preResponses` - same as `pre` but represented as the response object created by the pre method.
-- `query` - an object containing the query parameters.
+- `query` - an object containing the parsed [query parameters](https://nodejs.org/docs/latest/api/url.html#url_urlobject_query).
```

Comment:
Ah right, is this the ternary conditional on `typeof url === 'string'` here: https://github.com/hapijs/hapi/blob/770cc7bad15122f796d938738b7c05b66d2f4b7f/lib/request.js#L200-L201 ?